packages/flutter_local_notifications/README.md

```diff
@@ -0,0 +1,37 @@
+# flutter_local_notifications
+
+[![pub package](https://img.shields.io/pub/v/flutter_local_notifications_tizen.svg)](https://pub.dev/packages/flutter_local_notifications_tizen)
+
+The Tizen implementation of `flutter_local_notifications`.
```

Comment:
Is this true? Does `flutter_local_notifications_tizen` actually implement `flutter_local_notifications`? What's the relationship between the two packages?

Reply:
Strictly speaking `flutter_local_notifications_tizen` has no relationship with `flutter_local_notifications` other than api signature and functional similaries, because it's difficult to add Tizen in [NotificationDetails](https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/lib/src/notification_details.dart).
 This is similar to device_info_plus. I used it for consistency.

Reply:
Hmm. I'm not sure. `device_info_plus_tizen` has a very simple API set which is similar to other platforms' types and properties, but how much is common between `flutter_local_notifications` and your package? If there's not much similarity, wouldn't it be better to create your own package (something like `tizen_notifications`) which is self-contained, Tizen-only, and has a unique API set, just like `tizen_app_control`?

@bbrto21 How do you think?

Reply:
Hmm... It seems to me that this has no dependency on the original plugin and is a completely independent implementation. In my opinion, it would be good to change the package name and channel name to match the Tizen-only implementation.

Reply:
Thanks for the advice. I also think it would be better to implement it as a tizen only plugin. I will fix it.